caluma/caluma_workflow/api.py

```diff
@@ -39,3 +39,47 @@ def start_case(
     case = models.Case.objects.create(**validated_data)
 
     return domain_logic.StartCaseLogic.post_start(case, user, parent_work_item)
+
+
+def complete_work_item(work_item: models.WorkItem, user: BaseUser) -> models.WorkItem:
+    """
+    Complete a work item (just like `completeWorkItem`).
+
+    Usage example:
+    ```py
```

Comment:
In python docstrings, examples are usually denoted not in markdown syntax, but using this:
```
>>> complete_work_item(
...     work_item=models.WorkItem.first(),
...     user=AnonymousUser()
... )
WorkItem object xyz
```
(Whereas the last line represents the return value). This is mostly how it would look when typed into the python interpreter.

Reply:
Could you attach a link to the documentation of that standard?

Reply:
https://docs.python-guide.org/writing/documentation/#docstrings-and-magic

It's more convention, as the `doctest` module uses the same syntax - even though we don't actually use `doctest`

Reply:
Perfect thanks, I'll change it!

Reply:
Fixed